dask/dataframe/tests/test_io.py

```diff
@@ -57,7 +68,7 @@ def test_read_csv(open_comp_pair, infer):
         result = f.compute(get=dask.get)
         # index may be different
         assert eq(result.reset_index(drop=True),
-                  pd.read_csv(fn, compression=compression, lineterminator='\n'))
+                  pd.read_csv(io.StringIO(text), lineterminator='\n'))
```

Comment:
I wanted to test read_csv with xz compression, but only the pandas dev versions supports this. Perhaps I skip with pandas version <= 0.18.0 then.